tests/test_username_api.py

```diff
@@ -100,25 +120,38 @@ def test_available_username(self, website, user):
         expected = get_expected_response(website, user, 404)
         actual = get_response(self.app, website, user)
 
-        message = None
         if expected != actual:
             message = 'The provided available username ({}) returned 200'.format(
                 user)
-            username_pattern = self.websites['username_patterns'][website]
-            username_length = 15
+            logging.getLogger().warn(message)
```

Comment:
we dont need logging any more as the old test method is split, and each new test method has a single meaningful error